src/app/feed/blogpost/[id]/page.tsx

```
@@ -0,0 +1,109 @@
+                </p>
+              </div>
+            </Link>
+
+            <p className="category-tag">{post.category}</p>
+          </div>
+
+          <h3 className="text-30-bold">Pitch Details</h3>
+          {parsedContent ? (
+            <article
+              className="prose font-work-sans max-w-4xl break-all"
+              dangerouslySetInnerHTML={{ __html: parsedContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)